src/plots/plots.js

```diff
@@ -1494,6 +1495,13 @@ plots.supplyLayoutGlobalDefaults = function(layoutIn, layoutOut, formatObj) {
     coerce('title.y');
     coerce('title.yanchor');
 
+    coerce('title.subtitle.text', layoutOut._dfltTitle.plotsubtitle);
+    Lib.coerceFont(coerce, 'title.subtitle.font', font, {
+        overrideDflt: {
+            size: Math.round(fontSize * 1.2)
```

Comment:
I wonder if it would be better to base this on `title.font` - ie if you set `title.font` but not `subtitle.font`, you just get a smaller version of `title.font`?

Reply:
Fixed -- subtitle font size now defaults to 0.7 times the title font size.